libp2p/connmanager.nim

```diff
@@ -236,12 +237,14 @@ proc cleanupConn(c: ConnManager, conn: Connection) {.async.} =
 
   trace "Connection cleaned up", conn
 
-proc peerStartup(c: ConnManager, conn: Connection) {.async.} =
+proc onConnUpgraded(c: ConnManager, conn: Connection) {.async.} =
   try:
     trace "Triggering connect events", conn
+    conn.upgraded.complete()
```

Comment:
doesn't this need a nil check?

Reply:
Shouldn't it gets initialized in `Connection`, but we can add one for good measure.